enterprise/static/enterprise/enterprise_course_enrollment_page.js

```diff
@@ -0,0 +1,60 @@
+function focusableElementsSelector() {
+    return [
+        'a[href], area[href], input:not([disabled]), select:not([disabled]),',
+        'textarea:not([disabled]), button:not([disabled]), iframe, object, embed,',
+        '*[tabindex], *[contenteditable]'
+    ].join(' ');
+}
+function enableTabIndexingOn(containerSelector) {
+    var container = $(containerSelector),
+        focusableItems = container.find('*').filter(focusableElementsSelector()).filter(':visible');
+    container.attr('aria-hidden', 'false');
+    focusableItems.attr('tabindex', '0');
+}
+function disableTabIndexingOn(containerSelector) {
+    var container = $(containerSelector),
+    focusableItems = container.find('*').filter(focusableElementsSelector()).filter(':visible');
+    container.attr('aria-hidden', 'true');
+    focusableItems.attr('tabindex', '-1');
+}
```

Comment:
@replaceafill It might be worth DRYing out this pair of functions. Here is a rough sketch:

```js
function toggleTabIndexing(containerSelector, newState) {
    var container = $(containerSelector),
        focusableItems = container.find('*').filter(focusableElementsSelector()).filter(':visible'),
        ariaHidden = 'true',
        tabIndex = '-1';
    if (newState === 'enabled') {
        ariaHidden = 'false';
        tabIndex = '0';
    }
    container.attr('aria-hidden', ariaHidden);
    focusableItems.attr('tabindex', tabIndex);
}
function enableTabIndexingOn(containerSelector) {
    toggleTabIndexing(containerSelector, 'enabled');
}
function disableTabIndexingOn(containerSelector) {
    toggleTabIndexing(containerSelector, 'disabled');
}
```